bundles/binding/org.openhab.binding.ddwrt/OSGI-INF/genericbindingprovider.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2016 by the respective copyright holders.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.openhab.binding.ddwrt.genericbindingprovider">
+   <implementation class="org.openhab.binding.ddwrt.internal.ddwrtGenericBindingProvider"/>
```

Comment:
Please make sure all of your class and interface names start with a capital letter.